config/test_settings.py

```diff
@@ -0,0 +1,3 @@
+
+# the size of the vehicle's fuel tank
+FUEL_CAPACITY_US_GALLONS = 10.5
```

Comment:
I added a separate test file, because if I didn't, and we changed the fuel tank size the tests would fail.
So the test_settings file is meant to contain never changing values that we can test against